partiql-eval/src/main/kotlin/org/partiql/eval/internal/SqlCompiler.kt

```diff
@@ -0,0 +1,519 @@
+import org.partiql.plan.v1.operator.rel.RelCollation
+import org.partiql.plan.v1.operator.rel.RelDistinct
+import org.partiql.plan.v1.operator.rel.RelError
+import org.partiql.plan.v1.operator.rel.RelExcept
+import org.partiql.plan.v1.operator.rel.RelExclude
+import org.partiql.plan.v1.operator.rel.RelExcludeCollectionWildcard
+import org.partiql.plan.v1.operator.rel.RelExcludeIndex
+import org.partiql.plan.v1.operator.rel.RelExcludeKey
+import org.partiql.plan.v1.operator.rel.RelExcludePath
+import org.partiql.plan.v1.operator.rel.RelExcludeStep
+import org.partiql.plan.v1.operator.rel.RelExcludeStructWildcard
+import org.partiql.plan.v1.operator.rel.RelExcludeSymbol
+import org.partiql.plan.v1.operator.rel.RelFilter
+import org.partiql.plan.v1.operator.rel.RelIntersect
+import org.partiql.plan.v1.operator.rel.RelIterate
+import org.partiql.plan.v1.operator.rel.RelJoin
+import org.partiql.plan.v1.operator.rel.RelJoinType
+import org.partiql.plan.v1.operator.rel.RelLimit
+import org.partiql.plan.v1.operator.rel.RelOffset
+import org.partiql.plan.v1.operator.rel.RelProject
+import org.partiql.plan.v1.operator.rel.RelScan
+import org.partiql.plan.v1.operator.rel.RelSort
+import org.partiql.plan.v1.operator.rel.RelUnion
+import org.partiql.plan.v1.operator.rel.RelUnpivot
+import org.partiql.plan.v1.operator.rel.RelVisitor
+import org.partiql.plan.v1.operator.rex.Rex
+import org.partiql.plan.v1.operator.rex.RexArray
+import org.partiql.plan.v1.operator.rex.RexBag
+import org.partiql.plan.v1.operator.rex.RexCallDynamic
+import org.partiql.plan.v1.operator.rex.RexCallStatic
+import org.partiql.plan.v1.operator.rex.RexCase
+import org.partiql.plan.v1.operator.rex.RexCast
+import org.partiql.plan.v1.operator.rex.RexCoalesce
+import org.partiql.plan.v1.operator.rex.RexError
+import org.partiql.plan.v1.operator.rex.RexLit
+import org.partiql.plan.v1.operator.rex.RexMissing
+import org.partiql.plan.v1.operator.rex.RexNullIf
+import org.partiql.plan.v1.operator.rex.RexPathIndex
+import org.partiql.plan.v1.operator.rex.RexPathKey
+import org.partiql.plan.v1.operator.rex.RexPathSymbol
+import org.partiql.plan.v1.operator.rex.RexPivot
+import org.partiql.plan.v1.operator.rex.RexSelect
+import org.partiql.plan.v1.operator.rex.RexSpread
+import org.partiql.plan.v1.operator.rex.RexStruct
+import org.partiql.plan.v1.operator.rex.RexSubquery
+import org.partiql.plan.v1.operator.rex.RexSubqueryComp
+import org.partiql.plan.v1.operator.rex.RexSubqueryIn
+import org.partiql.plan.v1.operator.rex.RexSubqueryTest
+import org.partiql.plan.v1.operator.rex.RexTable
+import org.partiql.plan.v1.operator.rex.RexVar
+import org.partiql.plan.v1.operator.rex.RexVisitor
+import org.partiql.planner.catalog.Session
+import org.partiql.spi.fn.Agg
+import org.partiql.types.PType
+import org.partiql.plan.Rel as IRel
+
+/**
+ * See https://github.com/partiql/partiql-lang-kotlin/blob/v1/partiql-eval/src/main/kotlin/org/partiql/eval/internal/Compiler.kt
```

Comment:
Non-blocking: Since `v1` is dynamic, it'd be better to use the permalink: https://github.com/partiql/partiql-lang-kotlin/blob/122244e308d277630d36747806859d8a8a2153ff/partiql-eval/src/main/kotlin/org/partiql/eval/internal/Compiler.kt